exec/java-exec/src/main/resources/drill-module.conf

```diff
@@ -110,6 +110,7 @@ drill.exec: {
   http: {
     enabled: true,
     ssl_enabled: false,
+    cors_enabled: true,
```

Comment:
perhaps we should consider extending cors configuration with 
```
http: {
  cors: {
    enabled: true,
    allowedOrigins: ['*.mydomain.com', '*.someother.net'], --> configures Access-Control-Allow-Origin
    allowedMethods: ['option', 'get', 'post', 'some-other'], --> configures  Access-Control-Allow-Methods
    allowedHeaders: ['some-allowed-header'], --> configures Access-Control-Expose-Headers 
    credentials: true | false -- >  configures Access-Control-Allow-Credentials
  }
}
```
I am probably missing few in the list but I find these options essential.